### DISCUSSION TITLE
Home Assistant and Coral problems

### DISCUSSION THREAD

**Discussion by haw3000** [blakeblackshear/frigate#4081] [General]
Hi
I have a pi4 64 bit home assistant.
I have tried with success frigate with one camera and, satisfied, I have bought a Coral usb card.
I have add the following line sto the frigate.yml file.
detectors:
  coral:
    type: edgetpu
    device: usb
but I continue to receive the error
frigate.edgetpu                ERROR   : No EdgeTPU was detected
I have used a powered Hub Usb, but I have still the same reply
What I am doing wrong? What I could try?
Thanks

**Comment by NickM-27** [comment]
If you are running HA OS then see https://github.com/blakeblackshear/frigate/issues/3890 otherwise please provide your docker compose file or docker CLI
